README.md

```diff
@@ -9,117 +19,62 @@ This repository explores the application of XL-Net on user review based question
+Everything needed fot the environment
 
-- List all packages and software needed to build the environment
-- This could include cloud command line tools (i.e. gsutil), package managers (i.e. conda), etc.
+## Steps to run
 
-#### Dependencies
+### Step1: Configuration 
 
-- [Streamlit](streamlit.io)
+### Step2: Prepare and Preprocess
+#### - Download dataset
+Download the dataset you want to use for finetuning
+The datasets used in this project are:
+- **The [Squad dataset](https://rajpurkar.github.io/SQuAD-explorer/) is used in this proeject.**
+- **The manual sampled and labeled AmazonQA and preprocessed dataset is available [here]**
 
-#### Installation
-To install the package above, pleae run:
-```shell
-pip install -r requiremnts
-```
-
-## Build Environment
-- Include instructions of how to launch scripts in the build subfolder
-- Build scripts can include shell scripts or python setup.py files
-- The purpose of these scripts is to build a standalone environment, for running the code in this repository
-- The environment can be for local use, or for use in a cloud environment
-- If using for a cloud environment, commands could include CLI tools from a cloud provider (i.e. gsutil from Google Cloud Platform)
-```
-# Example
+#### - Download model checkpoints
+- The model checkpoints is available [here]()
 
-# Step 1
-# Step 2
+#### - Convert dataset to SQuAD format(Optional)
+If you want to try other dataset, it needs to be converted to SQuAD format first.
 ```
-
-## Configs
-- We recommond using either .yaml or .txt for your config files, not .json
-- **DO NOT STORE CREDENTIALS IN THE CONFIG DIRECTORY!!**
-- If credentials are needed, use environment variables or HashiCorp's [Vault](https://www.vaultproject.io/)
-
-
-## Test
-- Include instructions for how to run all tests after the software is installed
+code for converting to squad
 ```
-# Example
-
-# Step 1
-# Step 2
+#### - Preprocess data
 ```
-
-## Run Inference
-- Include instructions on how to run inference
-- i.e. image classification on a single image for a CNN deep learning project
 ```
-# Example
-
-# Step 1
-# Step 2
+### Step3: Train model
 ```
-
-## Build Model
-- Include instructions of how to build the model
-- This can be done either locally or on the cloud
+Bash scripts/tpu_run_squad.sh
 ```
-# Example
 
-# Step 1
-# Step 2
-```
+### Step4: Evaluate model
 
-## Serve Model
-- Include instructions of how to set up a REST or RPC endpoint
-- This is for running remote inference via a custom model
-```
-# Example
+### Step5: Inference model
 
-# Step 1
-# Step 2
-```
+### Step6: run the flask app on your local machine
 
 ## Analysis
-- Include some form of EDA (exploratory data analysis)
-- And/or include benchmarking of the model and results
-```
-# Example
 
-# Step 1
-# Step 2
-```
+#### Final result:
+
+Model | Finetune Dataset | Validation Dataset | AmazonQA Sample Coverage | EM | F1
+------|------------------|--------------------|--------------------------|----|---
```

Comment:
Consider changing the width of the columns accordingly. Here are some examples: https://stackoverflow.com/questions/36121672/set-table-column-width-via-markdown/51701842

Reply:
Not sure how to put css in the markdown?